### PR TITLE
Add markdown tabs

### PR DESCRIPTION
**Pull request by julieg18**
* I went ahead and converted this pull request to draft since I placed test markdown tabs in docs/index.md.

### CODE REVIEW

**Review by rogermparent — COMMENTED** [review]
The implementation logic is a little hard to follow and could possibly be more elegant, but it looks like it works really well such that I'd say it's more than good enough to let in as a first iteration!

**Review by casperdcl — APPROVED** [review]
good for me too as a first iteration :tada: (PS failing link checks were fixed in #110)

**Comment by julieg18** [comment]
> The implementation logic is a little hard to follow and could possibly be more elegant, but it looks like it works really well such that I'd say it's more than good enough to let in as a first iteration!

Agreed! I attempted to simplify as much as I could but it definitely still looks complex. Hopefully we can improve it at a later date!

**Comment by shcheklein** [comment]
Could you please clarify - does it render things server side or client side?

**Comment by julieg18** [comment]
> Could you please clarify - does it render things server side or client side?

I believe the tabs are rendered server side but input ids are rendered client side.

**Comment by shcheklein** [comment]
So, how much of DOM manipulation is happening on the client side? Before we have been trying hard to keep everything client side (to keep JS to minimum, and to keep page as prepare as it could be)

**Comment by julieg18** [comment]
> So, how much of DOM manipulation is happening on the client side? Before we have been trying hard to keep everything client side (to keep JS to minimum, and to keep page as prepare as it could be)

Hmmmmm, I think I misunderstood your original question. What did you mean by server side/client side? But as for DOM manipulation, I did my best to use the least amount of JS as I could. We use JS to keep track of the input checked state (so we can connect different sets of inputs) and to generate the IDs for the inputs. The rest is HTML and CSS.

**Comment by shcheklein** [comment]
Okay, no worries. I see that we use HTML to define section and markdown supports it as-is (no need to do any transformations client side markup->HTML). I think it should be fine. Sorry for the confustion.

**Comment by shcheklein** [comment]
@yalozhkin could you please take a look and suggest how to design this?  (not a blocker to merge)

**Review by rogermparent — APPROVED** [review]
Once the test data is removed, we can merge this first iteration!
